src/views/SystemIntake/Documents/UploadForm.tsx

```diff
@@ -268,6 +278,60 @@ const UploadForm = ({ type = 'requester' }: UploadFormProps) => {
             </Fieldset>
           </FormGroup>
 
+          {/* display for admins only when accessed from admin view */}
+          {type === 'admin' && user.isITGovAdmin(groups, flags) && (
+            <Controller
+              name="sendNotification"
+              control={control}
+              render={({ field, fieldState: { error } }) => (
+                <FormGroup className="margin-top-3" error={!!error}>
+                  <Label htmlFor={field.name}>
+                    {t(
+                      'technicalAssistance:documents.upload.sendNotificationToGRBReviewers.header'
+                    )}
+                  </Label>
+                  <HelpText className="margin-top-05">
+                    {t(
+                      'technicalAssistance:documents.upload.sendNotificationToGRBReviewers.info'
+                    )}
+                  </HelpText>
+
+                  {error && (
+                    <ErrorMessage
+                      name="sendNotification"
+                      errors={errors}
+                      message={t('technicalAssistance:errors.fillBlank')}
+                    />
+                  )}
+
+                  <Radio
```

Comment:
These radio fields need `inputRef={field.ref}` added. React Hook Forms needs the ref to focus the input on error.

Reply:
thanks! updated